doc/Command-Reference.md

```diff
@@ -3132,6 +3134,31 @@ Example of the module supporting target mode
   Target Mode set to 1
   ```
 
+## CMIS debug
+
+### CMIS debug loopback
+
+This command is the standard CMIS diagnostic control used for troubleshooting link and performance issues between the host switch and transceiver module.
+
+**sfputil debug loopback**
+
+- Usage:
+  ```
+  sfputil debug loopback [OPTIONS] PORT_NAME LOOPBACK_MODE
```

Comment:
What do you mean by "[options]" here?

Reply:
It is automatically added in the help message below.
![image](https://github.com/sonic-net/sonic-utilities/assets/11884092/e14e679f-d5b5-456a-a645-4a2160056cf1)
I can remove it since there is no specific OPTIONS needed